### PR TITLE
Feat/curvefs/cluster monitor

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
cicheck

File: curvefs/src/mds/fs_manager.cpp [hunk 1]
```diff
@@ -1179,5 +1180,29 @@ bool FsManager::FillVolumeInfo(common::Volume* volume) {
     return true;
 }
 
+void FsManager::UpdateFsUsage(const FsId2FsUsage& infos) {
+    for (auto const& fsUsage : infos) {
+        FsInfoWrapper wrapper;
```
Comment (h0hmj) [inline curvefs/src/mds/fs_manager.cpp:1185]: seems no need to persist usage in fsinfo

File: curvefs/monitor/grafana/provisioning/dashboards/cluster.jon [hunk 2]
```diff
@@ -0,0 +1,1014 @@
+{
```
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply curvefs/monitor/grafana/provisioning/dashboards/cluster.jon:1]: > filename typo cluster.jon -> cluster.json?

fix

**Comment by Cyber-SiKu** [comment]
cicheck

File: curvefs/src/mds/fs_manager.cpp (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/mds/fs_manager.cpp]: > seems no need to persist usage in fsinfo

fix

File: curvefs/proto/common.proto [hunk 3]
```diff
@@ -109,3 +109,7 @@ message CopysetInfo {
     required uint32 poolId = 1;
     required uint32 copysetId = 2;
 }
+
+message FsUsageInfo {
+    required uint64 used = 1;
```
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply curvefs/proto/common.proto]: > specific bytes in name

fix

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Pull request closed without merging**